lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -1898,13 +1899,21 @@ LogicalResult StmtEmitter::visitStmt(OutputOp op) {
   for (ModulePortInfo port : parent.getPorts()) {
     if (!port.isOutput())
       continue;
+
+    auto operand = op.getOperand(operandIndex);
+    if (operand.hasOneUse() && dyn_cast_or_null<InstanceOp>(operand.getDefiningOp())) {
+      ++operandIndex;
+      ++numStatementsEmitted;
```

Comment:
Please don't increment numStatementsEmitted.  It doesn't really matter here, but this should track the number of verilog statements emitted and this isn't doing it.